janus_core/calculations/md.py

```diff
@@ -992,30 +992,30 @@ def __init__(
             **ensemble_kwargs,
         )
 
-    def get_log_stats(self) -> str:
+    def get_stats(self) -> str:
         """
-        Get thermodynamical statistics to be written to molecular dynamics log.
+        Get thermodynamical statistics to be written to file.
 
         Returns
         -------
         str
             Thermodynamical statistics to be written out.
         """
-        log_stats = MolecularDynamics.get_log_stats(self)
-        return log_stats + f" {self.temp}"
+        stats = MolecularDynamics.get_stats(self)
+        return stats + f" {self.temp}"
```

Comment:
Feels like there should be a nicer way to do this like a dict of {name: prop} which is iterated over to build the headers/values, but again, not for this PR.